inject/inject.vala

```diff
@@ -508,6 +573,56 @@ namespace Frida.Inject {
 			return true;
 		}
 
+		/**
+		 * The script can send strings to frida-inject to write to its stdout or
+		 * stderr. This can be done either inside the RPC handler for receiving
+		 * input from frida-inject, or elsewhere at any arbitrary point in the
+		 * script. We use the follow syntax:
+		 *
+		 * send(['frida:stdout', 'MESSAGE']);
+		 * send(['frida:stderr', 'MESSAGE']);
+		 *
+		 * The resulting message will look as shown below. Note that we don't
+		 * use the parents object's `type` field since this is reserved for use
+		 * by the runtime itself.
+		 *
+		 * {"type":"send","payload":["frida:stdout","MESSAGE"]}
+		 */
+		private bool try_handle_stdout_message (Json.Object message) {
+			var payload = message.get_array_member ("payload");
```

Comment:
This API requires it to actually be an array if it exists. Might be easier to use a `JsonReader`.